libs/common/src/constants/portal/deal-status.ts

```diff
@@ -15,4 +15,5 @@ export const DEAL_STATUS = {
   COMPLETED: 'Completed',
   EXPIRED: 'Expired',
   WITHDRAWN: 'Withdrawn',
+  CANCELLED: 'Cancelled',
```

Comment:
No appropriate file to add this comment - can we add an E2E test?

E.g we have this already in the UKEF E2E suite`submit-AIN-deal-updates-portal-deal-status.spec.js`

Reply:
Will do so in a future ticket to reduce the scope of this PR (https://ukef-dtfs.atlassian.net/browse/DTFS2-7600 & https://ukef-dtfs.atlassian.net/browse/DTFS2-7601)